azure-pipelines.yaml

```diff
@@ -10,66 +10,69 @@ parameters:
     displayName: Component to Run
     type: object
     default:
-      - deployment: 'backendappgateway'
-        environment: 'sbox'
-        dependsOn: 'Precheck'
-      - deployment: 'backendappgateway'
-        environment: 'dev'
-        dependsOn: 'Precheck'
-      - deployment: 'backendappgateway'
-        environment: 'stg'
-        dependsOn: 'Precheck'
-      - deployment: 'backendappgateway'
-        environment: 'ithc'
-        dependsOn: 'Precheck'
-      - deployment: 'backendappgateway'
-        environment: 'test'
-        dependsOn: 'Precheck'
-      - deployment: 'backendappgateway'
-        environment: 'prod'
-        dependsOn: 'Precheck'
-      - deployment: 'appgateway'
-        environment: 'sbox'
-        dependsOn: 'Precheck'
-      - deployment: 'appgateway'
-        environment: 'dev'
-        dependsOn: 'Precheck'
-      - deployment: 'appgateway'
-        environment: 'stg'
-        dependsOn: 'Precheck'
-      - deployment: 'appgateway'
-        environment: 'ithc'
-        dependsOn: 'Precheck'
-      - deployment: 'appgateway'
-        environment: 'test'
-        dependsOn: 'Precheck'
-      - deployment: 'appgateway'
-        environment: 'prod'
-        dependsOn: 'Precheck'
+      - deployment: "backendappgateway"
```

Comment:
did you run some auto-formatter over this? I can't see any reason for any changes in this file?

Reply:
Yes, it's linting on save. I can revert it if you'd prefer?

Reply:
Reverted linting.